src/layer/annotation/index.ts

```diff
@@ -812,9 +828,53 @@ class RenderingOptionsTab extends Tab {
         ),
       ).element,
     );
+
+    const colorControlToolWidget = element.appendChild(
+      addLayerControlToOptionsTab(
+        this,
+        layer,
+        this.visibility,
+        LAYER_CONTROLS[ANNOTATION_COLOR_JSON_KEY],
+      ),
+    );
+    this.registerDisposer(
+      observeWatchable(
+        (hasDefaultColor) => {
+          if (hasDefaultColor) {
+            colorControlToolWidget.style.display = "";
+          } else {
+            colorControlToolWidget.style.display = "none";
+            this.layer.toolBinder.removeJsonString(ANNOTATION_COLOR_JSON_KEY);
+          }
+        },
+        makeCachedLazyDerivedWatchableValue(
+          (shader) => shader.match(/\bdefaultColor\b/) !== null,
+          layer.annotationDisplayState.shaderControls.processedFragmentMain,
+        ),
+      ),
+    );
   }
 }
 
+const LAYER_CONTROLS: Record<
+  string,
+  LayerControlDefinition<AnnotationUserLayer>
+> = {
+  [ANNOTATION_COLOR_JSON_KEY]: {
+    label: "Annotation color",
+    title:
+      "Annotation shader default color (enabled with 'defaultColor()' in the shader code)",
+    toolJson: ANNOTATION_COLOR_JSON_KEY,
+    ...colorLayerControl(
```

Comment:
The isValid property can be specified here instead of the `hasDefaultColor` logic above.